app/assets/stylesheets/fae/modules/forms/_label.scss

```diff
@@ -18,4 +18,19 @@ label {
     font-style: italic;
     line-height: 1.3;
   }
+
+  &.label--two_col {
+    display: flex;
+
+    .label_inner {
+      flex-shrink: 0;
+      max-width: 50%;
```

Comment:
Would prefer `flex: 0 0 50%` instead, if flex is determined to be supported (see previous comment)